src/courtroom.cpp

```diff
@@ -1347,12 +1350,15 @@ void Courtroom::handle_chatmessage(QStringList *p_contents)
       ui_vp_objection->play("takethat", f_char, f_custom_theme);
       objection_player->play("takethat.wav", f_char, f_custom_theme);
       break;
-    //case 4 is AO2 only
-    case 4:
-      ui_vp_objection->play("custom", f_char, f_custom_theme);
-      objection_player->play("custom.wav", f_char, f_custom_theme);
-      break;
     default:
+      if(objection_mod > 40) //checks if there is custom objection as it starts from code 40 and up
```

Comment:
Why 40? Why not keep `4` to be designated as `custom` for characters who use the old format, and then have any other free-form string to be the name of a custom objection sound?

Reply:
I set it to 40 as to be more clear that it's a custom objection. 
E.g 41 -> 4 for custom objection and 1 for the first one.
I considered using a free form string but it seemed like using the objections flag should be simple enough to do the job